bun/Dockerfile

```diff
@@ -23,13 +23,8 @@ RUN mkdir -p /etc/apt/keyrings \
 
 USER dependabot
 
-# Configure Node to use our custom CA bundle
+# Configure Bun to use our custom CA bundle
 ENV NODE_EXTRA_CA_CERTS=/etc/ssl/certs/ca-certificates.crt
```

Comment:
I understand that this build is for the  _Bun ecosystem_ but this comment is not accurate. This environment property is not used by Bun, but by Node (that bun depends on) 

_Perhaps this differentiation is pedantic, but it may cause confusion otherwise_